kivy/modules/inspector.py

```diff
@@ -530,7 +531,10 @@ def show_widget_info(self):
         keys = list(widget.properties().keys())
         keys.sort()
         node = None
-        wk_widget = weakref.ref(widget)
+        if type(widget) is WeakProxy:
+            wk_widget = widget.__ref__
```

Comment:
Shouldn't this just be `wk_widget = widget`? Because it's dereferenced by calling it, the same as `__ref__` and `__ref__` relies on internal details.

Reply:
WeakProxy doesn't seem to be callable...?

Reply:
Opps, I was thinking of `WeakMethod`. My bad.